lib/fatal_error/Kconfig

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig RESET_ON_FATAL_ERROR
+	bool "Reset on fatal error"
+	default y if !DEBUG && !KERNEL_DEBUG
+	help
+	  Enable using the fatal error handler defined for Nordic DKs.
+	  When it is used the system will be restarted after fatal error.
```

Comment:
```suggestion
	  When it is used, the system restarts after a fatal error.
```

Reply:
Ok changed

Reply:
I can't see the change, but that's probably GitHub messing up. ;)

Reply:
@KAGA164 this change does not seem to be implemented, no?